Automation.sc

```diff
@@ -18,23 +18,24 @@
 
 Automation {
     var <>length,
-        <>presetDir,
-        <>onPlay, <>onStop, <>onSeek,
-        <>playLatency = 0.1, <>seekLatency = 0.1,
-        <>doStopOnSeek = false, 
-        <>onEnd,
-        <>verbose = true,
-        <>server,
-        <>doDefer = true,
-        <doRecord = false,
-        <clients,
-        <>gui,
-        startTime = -1, startOffset = 0,
-        semaphore = nil,
-        playRoutine,
-        playStatus = false,
-        playDoReschedule = true,
-        playLastStopped = 0;
+	<>presetDir,
```

Comment:
It looks like you changed the indenting of many lines without actually modifying them. Please don't change the whitespace.